test/plugins/scenarios/fasthttp/config/excepted.yml

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+  - serviceName: fasthttp
+    segmentSize: ge 3
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: GET:http://service:8080/health
```

Comment:
We don't need to check `health`.

Reply:
This error occurred again in the kratosv2 test

actual: 
  Segment[20e6703a6e1d11eeb1e80242ac120003.29.39094317935270002] e
  expected:	Span[-1, 0] /helloworld.Greeter/SayHello
  actual:	span[-1, 0] GET:/health
  reason:	[operation name]: expected=>{/helloworld.Greeter/SayHello}, actual=>{GET:/health}

Reply:
`/health` should not exist, that is first. Then, that test may fail due to something inside that plugin or test case not stable.

Reply:
All right, I got it